src/assets/styles/config/breakpoints/project-about-modal-breakpoints.css

```diff
@@ -0,0 +1,60 @@
+@media (min-width: 768px) {
+  .modal {
+    padding-top: 3.5rem;
+    overflow-y: hidden;
+  }
```

Comment:
Hi @abdulhamiid! Thank you for your review as well as for such great feedback and fix pointers! 😃 🎯 
You are right! It makes sense to remove the hidden value, I will fix the issues right away!
Note:
As for the **issues**, it will point to the issues page once this branch is merged to 'main', at the moment it is giving you 404 due to its relative path.